### DISCUSSION TITLE
Designers Needed

### DISCUSSION THREAD

**Discussion by praveenscience** [praveenscience/Internship-LMS-FrontEnd#48] [Ideas]
We are planning to get our ideas into Bootstrap Design. We need designers who might help us in the UX part! Who's up for it? **Please write your experience in a nice elevator pitch** and let's get started. So first week will be about designs and setting up the project for #DevTernShip! 😄 Let's rock! 🤘🏻

**Comment by Chadha93** [comment]
Yes I can work over it

**Reply by praveenscience** [reply]
Please write your experience in a nice elevator pitch

**Comment by Komal-99** [comment]
I am in for giving my  best to it👍
And to learn a lot from here about UX designing.

**Comment by thebardenbella** [comment]
I am quite determined to work on my first open source project and hence would try to keep my involvement in the project as high as my determination

I have my hands on experience in HTML, CSS, JS , Bootstrap , and am in process of learning React :)

**Comment by ayush461** [comment]
SIR, I would love to contribute to your project and gain experience in open source development!!

**Comment by Deepak-dex** [comment]
###  I'm up for it ✌

Not a good or experienced ux/ui designer but I can give it a shot !

**Comment by NidhiSingh0068** [comment]
Up for it.
So, I have some experience with HTML5, CSS/SCSS, Bootstrap4, and Angular to create UI parts with responsive behavior. Currently learning ReactJs. Would like to work on UI for this project. I can create the web-designs according to the blueprint.

**Comment by shreya-gb** [comment]
Ready to boost up my share of skills with this engaging requirement alongside open contributions!! 🏆

**Comment by Ayushisood** [comment]
Hii @praveenscience , I would  like to work on it , since  with this opportunity I can  also experience the open source learning. I am also very much familiar with bootstrap , HTML5 and CSS

**Comment by aashi2912** [comment]
I have some experience with bootstrap and can do contribution using HTML, CSS and bootstrap on designing the website! Done small project using Bootstrap.

Here is the link for demo video : https://drive.google.com/file/d/1rEmFC42W1EEaI4eIeSwnPdvFjtwSPaMW/view?usp=sharing
GitHub : https://github.com/aashi2912/PerfectCup

Responsive website.

**Comment by EANimesha** [comment]
Up for it. 
I have experience in Bootstrap, CSS, material UI, Ant design, and also with JS frameworks(including Angular and ReactJS). 
I have experience in Figma and AdobeXD designing.
**Idea:** 
**We can first start the design with UI/UX design tools like Figma.** 
From that, we can brainstorm a perfect design. 
Check one of my basic Figma design [`here`](https://www.figma.com/file/dxudfnh1MZmDmDs1gVghLf/Task?node-id=0%3A1) 
I am interested to contribute to the design.

**Comment by Ankita297** [comment]
Sir I would like to contribute to this project . I have experience in html ,css and bootstrap .  
I would love to contribute to this project.

**Comment by vaishnavig20** [comment]
I would love to contribute

**Reply by vaishnavig20** [reply]
I have experience iin using Bootstrap.,Css,Html and also React.
I also have knowledge of figma and AdobeXD

**Comment by pm0824** [comment]
Hi, I am Prachi
I would like to contribute to the design. 
I have experience in developing the UI using **CSS, Sass and Bootstrap**. 
Checkout one of the website recreation [`here`](https://pm0824.github.io/chefdiary)
I have also learnt the UI/UX designing. I am not that much experienced but I can give my best.
Checkout one of my **Figma** design [`here`](https://www.figma.com/file/ewdseqN2OYqbxKfWi64yGU/Password-Strength-Checker?node-id=0%3A1)

**Comment by salma2vec** [comment]
Hi, I'm Salma.
I would like to contribute to this project. I have hands-on experience with *HTML5, CSS/SCSS, Javascript, and Bootstrap for responsive web design*. I have done few projects and I am currently working on some too (a WebApp).
I aim to leverage this opportunity to augment my skills in UI/UX designing.

**Comment by vedant115** [comment]
Sir I would like to contribute to this project and work under your guidance. It will be great learning experience for me. 
I have some experience in HTML, CSS, Bootstrap, React and react-bootstrap.
I would love to contribute to this project.

**Comment by Fab-Gaurav** [comment]
Sir I have experience in HTML , CSS and bootstrap .
I would love to contribute to this project.
@praveenscience

**Comment by sonalsingh25** [comment]
Hi I am interested in contributing to this. I don't have any experience in UI/UX but I am very much comfortable with HTML, CSS and Bootstrap. You can check one of my bootstrap work here https://sonal251001.github.io/PawFect/ . And also I would love to learn from your live videos if you conduct any and then contribute. Thanks :)

**Comment by sureandhar** [comment]
Hi from sureandhar,
I'm waiting for long time to contribute my first open-source project and glad to work under your guidance. I have some sort of experience with HTML, CSS and JS then I'm on my way to bootstrap and React. Done some small small projects.
Finally,.... **I'm up for !t...^ ÷ × ^**

**Comment by balarka135** [comment]
Hello, I am Balarka. I have experience in Bootstrap and React(level;-basic). This is my Github profile [https://github.com/balarka135](url).
I like to contribute to this project. Thanks in advance.

**Comment by ishani02** [comment]
Hello I am Ishani,
I am new to open source contribution but I am an avid learner and hard-woking. I am skilled and have hands-on experience in HTML,CSS,JavaScript and Bootstrap. I am also familiar with React(basic level) and React-bootstrap.Really looking forward to contributing to this project and also learning new skills.

**Comment by priyanshi220** [comment]
Hello, My name is Priyanshi
I am well experienced with HTML, CSS, Bootstrap, and also have a basic level of knowledge about JS and React JS.
Really looking forward to working on this project.

**Comment by pavankumaretta** [comment]
Hey !! 
I am Pavan Kumar very much interested and dedicated to web development and have done small works for UX/UI designing but have fluent efficiency through full stack development. I am very excited to contribute to this project It will be great learning experience for me. Thanks !!

**Comment by Himika-Mishra** [comment]
Hi, 

I am Himika Mishra and I am very much interested in this project especially in the UI part since I have a good knowledge of html5 css3 and bootstrap 4 too and have a basic knowledge about JS and ReactJS. I wanted to do some hands-on and improve my skills and this is the best opportunity for me. 
As a part of GSSOC'21 participant I am very much interested in contributing to this project and be a part of it. 
Please let me know @praveenscience .

Thanks 
Himika Mishra

**Comment by kaffan** [comment]
Hello sir this is khan affan Sibgatullah 
i have a hands on experience on HTML, CSS, Bootstrap and Vanilla.js and i have made multiple mini projects

**Comment by shriya030** [comment]
Hi, 
I am Shriya and I am very much interested in the designing part. I have a good knowledge of HTML, CSS, bootstrap, and basic knowledge of JS and React. I am looking forward to contributing to this project as it will give me hands-on experience and will enhance my skills.

**Comment by manasvvi1** [comment]
Hey,
I am Manasvvi, participant at GSSoc21. I have knowledge about HTML and CSS. I am quite interested in designing. I have made one site as of now (with 4 other people) for a hackathon i participated in recently. Looking forward to learn and contribute in the project! My first project link: https://github.com/manasvvi1/MentalHealth-Website.git

**Comment by Spring-dot** [comment]
Hey!
I am Akanksha Priya, a participant at GSSoC'21, 2nd year @IITR. I have experience in Figma(made wireframes for my project proposal for Goldman Sachs women summit as well as in hackathons like prospect100), bootstrap 4.0, CSS, html5. Also, I have worked with WordPress for my past intern in my first year, in which I made landing pages, education sites, and profile pages. I am up for working on the design as well as the development part. Last year I learned react for an open source project - contriFOSS for RGSoC. 
Let me know, how I can make useful contributions.
Thanks
